### ISSUE TITLE
views_plugin_style_grid_bootstrap

### ISSUE THREAD

**Issue by soniktrooth** [drupalprojects/kalatheme#267]
Do we need this for D8? If the functionality is desired we should check that there isn't something in contrib that doesn't already do this.

**Comment by andrewmallis** [comment]
I would suggest not doing it seeing if it is missed.

**Comment by soniktrooth** [comment]
Good call—also, if we're going framework agnostic then this would be a lot more abstracted if it has to cater for any grid system.

Shelving for now.

**Comment by andrewmallis** [comment]
@soniktrooth we should move this our of the beta milestone. Our alt is 1.0 at this time. Do we just remove the milestone?
